src/common/checksum.c

```diff
@@ -1371,3 +1641,32 @@ dcf_corrupt(d_sg_list_t *sgls, uint32_t nr)
 		((char *)sgl->sg_iovs->iov_buf)[0] += 2;
 	}
 }
+
+/** should probably go somewhere else */
+int
+dedup_get_csum_algo(struct dc_cont_props *cont_props)
+{
+	if (cont_props->dcp_dedup && cont_props->dcp_dedup_verify)
```

Comment:
(style)  'dedup' may be misspelled - perhaps 'dedupe'?
(style)  'dedup' may be misspelled - perhaps 'dedupe'?